src/app/frontend/replicationcontrollerdetail/serviceendpoint_directive.js

```diff
@@ -22,6 +22,7 @@ export default function serviceEndpointDirective() {
     templateUrl: 'replicationcontrollerdetail/serviceendpoint.html',
     scope: {
       'endpoint': '=',
+      'url': '@',
```

Comment:
This parameter should have a more descriptive name. Url does not convey what it actually is. You wouldn't know that this is a boolean value to display the endpoint as a link. How about `is-http` or `is-external-http` or something else to say that this is an externally available http endpoint. Using this info we can display this as a link.